### PR TITLE
Update Apache Server Configs to latest v2.13.0

### PR DESCRIPTION
**Pull request by drKraken**
Seems like need to update apache server config to the newest version as here https://github.com/h5bp/html5-boilerplate/commit/c0de8e873af6182f0d3461b149fd5d5b22aea7cd

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -85,7 +85,7 @@
 # Customize what Apache returns to the client in case of an error.
 # https://httpd.apache.org/docs/current/mod/core.html#errordocument
 
-ErrorDocument 404 /404.html
+# ErrorDocument 404 /404.html
```
Comment (arthurvr) [inline .htaccess]: For us, this line needs to stay in.

**Comment by drKraken** [comment]
@arthurvr ok, e5a1ebb085f8c1d9c2a419241230b08c43799028

File: .htaccess [hunk 2]
```diff
@@ -996,4 +995,4 @@ FileETag None
 #     RewriteEngine On
 #     RewriteCond %{REQUEST_FILENAME} !-f
 #     RewriteRule ^(.+)\.(\d+)\.(bmp|css|cur|gif|ico|jpe?g|js|png|svgz?|webp)$ $1.$3 [L]
-# </IfModule>
+# </IfModule>
```
Comment (arthurvr) [inline .htaccess]: missing newline

Comment (drKraken) [inline .htaccess]: oh, fixed it

**Pull request closed without merging**